tach.yml

```diff
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/gauge-sh/tach/v0.6.9/public/tach-yml-schema.json
+# yaml-language-server: $schema=https://raw.githubusercontent.com/gauge-sh/tach/v0.8.2/public/tach-yml-schema.json
```

Comment:
The public schema also needs to be updated with `forbid_circular_dependencies`! You can find it at `public/tach-yml-schema.json`